### ISSUE TITLE
Training settings in phase1 and phase 2

### ISSUE THREAD

**Issue by kuang23** [SangHunHan92/2K2K#18]
Hi,
Thanks for sharing your great work!
As mentioneed in paper, the learning rate, epoch, and batch size are 1e-4, 30, and 2. So do you use the same setting in phase1 and phase2 ?

**Comment by SangHunHan92** [comment]
I set same parameteres for both phase, but it seems phase2 converge more fast (About 10 epochs).
